jira7/jira_types.bal

```diff
@@ -336,24 +338,22 @@ public type IssueRequest record {
 # + statusId - Issue status Id
 # + issueType - Type of the jira issue
 public type IssueSummary record {
-    string resource_path;
-    string id;
-    string key;
-    string priorityId;
-    string statusId;
-    IssueType issueType;
+    string resource_path = "";
+    string id = "";
+    string key = "";
+    string priorityId = "";
+    string statusId = "";
+    IssueType issueType = {};
 };
 
 # Represent Jira Connector based errors.
 # + type - Type of the error (HTTP error,server error etc.)
 # + message - Error message
 # + jiraServerErrorLog - Error log returned by the jira server, for "server error" type
-# + cause - Cause for the error
 public type JiraConnectorError record {
```

Comment:
Please remove this record and use ballerina `error`.

Reply:
Do we need to keep this record for the error?

Reply:
I have created an issue to track this[1]

[1] https://github.com/wso2-ballerina/module-jira/issues/49